<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CountRiceWindow</class>
 <widget class="QMainWindow" name="CountRiceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>找米粒</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>打开图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="countRice">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>找米粒</string>
    </property>
   </widget>
   <widget class="QLabel" name="riceNum">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>米粒数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="origionImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>请打开需要处理的图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnMain">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>返回主页</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultImage">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
